source/tests/system/nirfmxnr_driver_api_tests.cpp

```diff
@@ -68,7 +68,7 @@ InitializeResponse init(const client::StubPtr& stub, const std::string& model)
 TEST_F(NiRFmxNRDriverApiTests, Init_Close_Succeeds)
```

Comment:
I pulled this down and ran and passed all RFmxNR system tests `.\SystemTestsRunner.exe --gtest_filter=*RFmxNR*`. Not sure if you're set up for that.

Reply:
Thanks! No, I'm not set up for it.